flink-dist/pom.xml

```diff
@@ -305,6 +305,7 @@ under the License.
 				<!--Build uber jar-->
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-shade-plugin</artifactId>
+				<version>2.4.1</version>
```

Comment:
Oh I didn't catch the version tag is intentionally removed. I'll update this except quickstart pom.